pkg/streamer/streamer.go

```diff
@@ -0,0 +1,7 @@
+package streamer
+
+import "github.com/operator-framework/operator-registry/pkg/api"
+
+type Stream interface {
+	Send(*api.Bundle) error
+}
```

Comment:
```suggestion
type BundleSender interface {
	Send(*api.Bundle) error
}
```

I'm sure this is debatable, but I would like to see the interface name be oriented to the behavior (sending Bundles somewhere). Stream is similar, but it invites confusion with gRPC "streams" and doesn't specify direction (can I receive from a Stream?).

Reply:
Addressed in the latest version of the PR.